plugins/out_stackdriver/stackdriver.c

```diff
@@ -2898,10 +2909,10 @@ static void cb_stackdriver_flush(struct flb_event_chunk *event_chunk,
 
     if (ret_code == FLB_RETRY) {
         update_retry_metric(ctx, event_chunk, ts, c->resp.status);
-    } else {
-        /* Update metrics counter by using labels/http status code */
-        update_http_metrics(ctx, event_chunk, ts, c->resp.status);
     }
+
+    /* Update metrics counter by using labels/http status code */
+    update_http_metrics(ctx, event_chunk, ts, c->resp.status);
```

Comment:
Wait, wouldn't we be multi-counting http metrics for retryable errors (5xx)?

Reply:
We are still making the request a second time.